src/main/java/org/incubyte/actors/PeopleService.java

```diff
@@ -11,7 +11,7 @@
 import java.util.Optional;
 @Singleton
 public class PeopleService {
-    private TmbdClient tmbdClient;
+    private final TmbdClient tmbdClient;
```

Comment:
Why is this removed?

Reply:
Hello, this is added because the value of tmdbClient need not be changed anywhere except  the constructor